grunttasks/eslint.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+'use strict';
+
+module.exports = function (grunt) {
+  grunt.config('eslint', {
```

Comment:
@TDA we need to run this task in https://github.com/mozilla/fxa-content-server/blob/master/grunttasks/lint.js#L10